docs/key-issues/multiple_architectures.md

```diff
@@ -0,0 +1,365 @@
+# Platforms with multiple CPU architectures
+
+One important subset of ABI concerns is the CPU architecture for which a binary
+artefact has been built. Attempting to run a binary on hardware that doesn't
+match the CPU architecture (or architecture variant[^1]) for which the binary
+was built will generally lead to crashes, even if the ABI being used is
+otherwise compatible.
+
+[^1]:
+    E.g., the x86-64 architecture has a range of well-known extensions, such as
+    SSE, SSE2, SSE3, AVX, AVX2, AVX512, etc.
+
+## Current state
+
+Most operating systems support multiple CPU architectures:
+
+* In the early days of Windows NT, both x86 and DEC Alpha CPUs were supported
+* Windows 10 supports x86, x86-64, ARMv7 and ARM64; Windows 11 supports x86-64
+  and ARM64.
+* Due to its open source nature, Linux tends to support all CPU architectures for
+  which someone is interested enough to author & provide support in the kernel,
+  see [here](https://en.wikipedia.org/wiki/List_of_Linux-supported_computer_architectures).
```

Comment:
Could you use an informative link label instead of `here`?

(there are other cases of this problem we can see in this PR, but good to start somewhere 🙂)

(refs https://www.w3.org/QA/Tips/noClickHere – https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/ – https://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/ etc)